index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <div class="toolbar">
+    <header>
+      <a class="my-logo" href="#">
+        <img src="/images/My-Logo.png" alt="My Logo icon.">
+      </a>
+    </header>
+      <nav>
+        <a class="hamburger" href="#">
+          <img src="/images/Hamburger.png" alt="Hamburger icon.">
+        </a>
+      </nav>
+    </div>
+      <div class="background">
+    <main class="headline">
+      <h2 class="primarytext">I’m Marcos
+        Glad to see you!</h2>
+        <article class="primarytext2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</article>
+        <a class="connect" href="#">
+          <img src="/images/connect.png" alt="LET'S CONNECT icon">
+        </a>
+    </main>
+    <div class="social">
+    <a class="connect" href="#">
+      <img src="/images/Twitter.png" alt="Twitter icon">
+    </a>
+    <a class="connect" href="#">
+      <img src="/images/Linkedin-icon.png" alt="Linkedin icon">
+    </a>
+    <a class="connect" href="#">
+      <img src="/images/Mail.png" alt="Mail icon">
+    </a>
+    <a class="connect" href="#">
+      <img src="/images/Git.png" alt="Git icon">
+    </a>
+    <a class="connect" href="#">
+      <img src="/images/Peace.png" alt="Peace icon">
+    </a>
+      </div>
```

Comment:
- [ ] It would be semantically better if the list of icons have a `<ul>` tag grouping them, and each icon should be a list-item `<li>`. 
- [ ] The icon images are broken. Kindly add a `./` to the `img src` path as I showed you in an earlier suggestion. See below 👇 
![image](https://user-images.githubusercontent.com/46839548/186222800-38b0ea4f-bf53-43b9-abc0-952e6d50b7bb.png)